### PR TITLE
Translate library/string.rst line 94

### PR DESCRIPTION
**Pull request by ShamrockLee**
This PR contains the Taiwanese Traditional Chinese (zh-TW) translation of [CPython v3.12 `library/string.rst:94`](https://github.com/python/cpython/blob/3.12/Doc/library/string.rst?plain=1#L94).  

### Questions

- What is the idiomatic translation of "this function/method takes ...", is it "這個 function/method 的引數包括……", or "這個 function/method 接收……"?
- How do we translate "It's a wrapper [of ...]"?

### CODE REVIEW

File: library/string.po [hunk 1]
```diff
@@ -120,6 +120,8 @@ msgid ""
 "positional and keyword arguments. It is just a wrapper that calls :meth:"
 "`vformat`."
 msgstr ""
+"主要的 API method。其引數包括一個格式字串 (:ref:`formatstrings`) 及一組任意的"
+"位置引數與關鍵字引數。這個 method 包裝了 :meth:`vformat`。"
```
Comment (ken71301) [inline library/string.po]: wrapper 可翻為 包裝器

**Comment by rockleona** [comment]
hi @ShamrockLee ,

to your quetions:
- I think 「接收」 will be better
- I think "wrapper" can stay in english, since we might not use it in Traditional Chinese frequently

File: library/string.po [hunk 2]
```diff
@@ -120,6 +120,8 @@ msgid ""
 "positional and keyword arguments. It is just a wrapper that calls :meth:"
 "`vformat`."
 msgstr ""
+"主要的 API method。它接收一個格式字串及一組任意的"
+"位置引數與關鍵字引數，是 :meth:`vformat` 的 wrapper。"
```
Comment (ken71301) [inline library/string.po]: 這邊的 calls 沒翻到

File: library/string.po (hunk 1 above)
Comment (ken71301) [reply library/string.po]: 我看到現有的翻譯中 wrapper 有被翻譯為 包裝器 或 包裝函式
我們可能得開個 discussion 統一一下了

**Comment by mattwang44** [comment]
rebase

**Comment by ShamrockLee** [comment]
Sorry for the delay. I was busy working on an assignment recently.

I'll address the review suggestions before next sunday.

**Comment by mattwang44** [comment]
@ShamrockLee 
有時間再處理就好，別感到有壓力 👍🏽

**Comment by mattwang44** [comment]
@ShamrockLee 
最近要 wrap up 我們 11 月活動的所有 PR 了，想請問你有空來處理這個 PR 嗎？
還是我能否來幫你修正剩下未完成的部分呢？

**Comment by ShamrockLee** [comment]
今天期末剛結束。我處理個。

**Comment by ShamrockLee** [comment]
Rebased on top of event-1125 and resolved merge conflict.

File: library/string.po (hunk 2 above)
Comment (ShamrockLee) [reply library/string.po]: 已加入「呼叫」。

**Review by ken71301 — APPROVED** [review]
LGTM~

**Pull request merged**